core/domain/feedback_services_test.py

```diff
@@ -772,22 +773,20 @@ def test_that_reply_to_id_is_created(self):
             # Check that reply_to id is created for user A.
             model = email_models.GeneralFeedbackEmailReplyToIdModel.get(
                 self.user_id_a, thread_id)
-            cross_model = (
-                email_models.GeneralFeedbackEmailReplyToIdModel
-                .get_by_reply_to_id(model.reply_to_id))
-            self.assertEqual(model, cross_model)
-            self.assertEqual(cross_model.user_id, self.user_id_a)
+            cross_object = email_services.get_general_feedback_reply_to_id(
```

Comment:
Don't know what a cross_object is (or a cross_model in the old version of the code, for that matter :P)

I'm somewhat uncertain of what this is supposed to be testing, in the revised version of the code. Seems like this file should test the services layer, so might not need to reference email_models at all. Just check that the fields of cross_object (which should probably be renamed) match the expected values.